price_feeds_poller.py

```diff
@@ -26,57 +30,77 @@ def handle_requestUpdate(
   ):
 
     try:
+      print(f" - WitnetRequestBoard: {contract.functions.witnet().call()}")
+      print(f" - WitnetPriceRouter:  {router.address}")
+      print(f" - WitnetPriceFeed:    {contract.address}")
+
       # Check that the account has enough balance
       balance = w3.eth.getBalance(network_from)
       if balance == 0:
           raise Exception("Master account run out of funds")
+      print(f" - Account       : {network_from}")        
+      print(f" - Balance       : {round(balance / 10 ** 18, 5)} {network_symbol}")
 
+      # Apply gas price strategy, if any
       if network_gas_price is None:
-        print(f"Estimating gas price from last blocks...")
         network_gas_price = w3.eth.generateGasPrice()
+      print( " - Tx. gas price :", "{:,}".format(network_gas_price))     
+      
+      if network_gas is not None:
+        print( " - Tx. gas limit :", "{:,}".format(network_gas))
```

Comment:
This could easily be moved inside the `else` clause below for simplicity

Reply:
this was done on purpose, as to force loging the "tx. gas limit" (if any) before the "tx. gas value"

aesthetics rationale: ease logs readability by putting two comma-justified big integer values in a row, while avoiding to print a comma/dotted currency in between